coop_shift/report/report_wallchart_common.py

```diff
@@ -52,24 +51,24 @@ def _get_tickets(
         return template.shift_ticket_ids.filtered(
             lambda t, s=self, p=product_name: t.product_id == s.env.ref(p))
 
-    @api.model
-    def prerender_html(self, data):
-        self.model = self.env.context.get('active_model')
-        docs = self.env[self.model].browse(self.env.context.get('active_id'))
+    # @api.model
+    # def prerender_html(self, data):
+    #     self.model = self.env.context.get('active_model')
+    #     docs = self.env[self.model].browse(self.env.context.get('active_id'))
 
-        docargs = {
-            'doc_ids': self.ids,
-            'partner_id': self.env.user.partner_id,
-            'doc_model': self.model,
-            'data': data['form'],
-            'docs': docs,
-            'date': date,
-            '_': _,
-        }
-        return docargs
+    #     docargs = {
+    #         'doc_ids': self.ids,
+    #         'partner_id': self.env.user.partner_id,
+    #         'doc_model': self.model,
+    #         'data': data['form'],
+    #         'docs': docs,
+    #         'date': date,
+    #         '_': _,
+    #     }
+    #     return docargs
 
-    @api.multi
-    def render_html(self, data):
-        docargs = self.prerender_html(data)
-        return self.env['report'].render(
-            'coop_shift.report_wallchart_common', docargs)
+    # @api.multi
+    # def render_html(self, data):
+    #     docargs = self.prerender_html(data)
+    #     return self.env['report'].render(
+    #         'coop_shift.report_wallchart_common', docargs)
```

Comment:
I see that you cleaned this report code. That's great. I have to say I'm not sure what you've done exactly, but we'll test it after its merged :)

Can you please remove the commented lines, though? There's no need to keep them, if they're no longer necessaries.
Are they, though? No longer necesarries